### ISSUE TITLE
results on MOT17 validation set in Table 7

### ISSUE THREAD

**Issue by liuqk3** [ifzhang/FairMOT#304]
Hi Yifu,

Thanks for your great work. Here I have an issue about the results in Table 7 (in the newer version paper).

The first row shows the results when only trained on MOT17 is 67.5% MOTA and 69.9% IDF1. I tried to reproduce these results.
I trained a model using `python train.py mot --exp_id mot17_half_dla34 --load_model '../models/ctdet_coco_dla_2x.pth' --data_cfg '../src/lib/cfg/mot17_half.json' `, and test the model_30.pth using `python track_half.py mot --load_model ../exp/mot/mot17_half_dla34/model_30.pth --conf_thres 0.4 --val_mot17 True`, the results are 60.7% MOTA, 67.3% IDF1, which are much lower than those presented in Table 7.

Do you have any suggestions about the results?
Thanks
@ifzhang

**Comment by ifzhang** [comment]
You can check whether you load the coco pretrained model correctly. When loading coco pretrained model and trained on half MOT17, you can get 67+ MOTA. If you train from scratch, you can get only 60+ MOTA.

**Comment by yangyangkiki** [comment]
Hi Yifu,
I have the same issue with @liuqk3. I try to reproduce the results of the first row in Table7.

I trained the model by using :
python train.py mot --exp_id mot17_dla34 --load_model '../models/ctdet_coco_dla_2x.pth' --data_cfg '../src/lib/cfg/mot17.json' --gpus 0,1 , 
and then test the model_30.pth by using : 
python track.py mot --load_model ../exp/mot/mot17_dla34/model_30.pth --conf_thres 0.4 --test_mot17 True .

Here is my problem:
1. I found @liuqk3 adopted the mot17_half.json file to train the model, but I choose mot17.json. I read your paper again, and I don't know the difference between these two files. can you help to explain this?

2. When I want to submit my results to MOT chanllenge server, I found that It should be many files in the .zip file on the instructions page, like this:

./MOT17-01-DPM.txt 
./MOT17-02-DPM.txt 
./MOT17-03-DPM.txt 
./MOT17-04-DPM.txt 
./MOT17-05-DPM.txt 
./MOT17-06-DPM.txt 
./MOT17-07-DPM.txt 
./MOT17-08-DPM.txt 
./MOT17-09-DPM.txt 
./MOT17-10-DPM.txt 
./MOT17-11-DPM.txt 
./MOT17-12-DPM.txt 
./MOT17-13-DPM.txt 
./MOT17-14-DPM.txt 
./MOT17-01-FRCNN.txt 
./MOT17-02-FRCNN.txt 
./MOT17-03-FRCNN.txt 
./MOT17-04-FRCNN.txt 
./MOT17-05-FRCNN.txt 
./MOT17-06-FRCNN.txt 
./MOT17-07-FRCNN.txt 
./MOT17-08-FRCNN.txt 
./MOT17-09-FRCNN.txt 
./MOT17-10-FRCNN.txt 
./MOT17-11-FRCNN.txt 
./MOT17-12-FRCNN.txt 
./MOT17-13-FRCNN.txt 
./MOT17-14-FRCNN.txt 
./MOT17-01-SDP.txt 
./MOT17-02-SDP.txt 
./MOT17-03-SDP.txt 
./MOT17-04-SDP.txt 
./MOT17-05-SDP.txt 
./MOT17-06-SDP.txt 
./MOT17-07-SDP.txt 
./MOT17-08-SDP.txt 
./MOT17-09-SDP.txt 
./MOT17-10-SDP.txt 
./MOT17-11-SDP.txt 
./MOT17-12-SDP.txt 
./MOT17-13-SDP.txt 
./MOT17-14-SDP.txt

But  there are just 7 files saved in the results folder when I finished running the track.py file:
./MOT17-01-SDP.txt 
./MOT17-03-SDP.txt 
./MOT17-06-SDP.txt 
./MOT17-07-SDP.txt 
./MOT17-08-SDP.txt 
./MOT17-12-SDP.txt 
./MOT17-14-SDP.txt

Can you give me some suggestions?

Thank you so much.
@ifzhang

**Comment by ifzhang** [comment]
mot17.json contains all the training images in mot17 and mot17_half.json contains half of training images of mot17 (for each sequence, first half training, second half evaluation).

**Comment by Wenbin94** [comment]
> 
Have you solved this problem? My test results are similar to yours. MOTA  60.7,IDF1 69.1
